package.json

```diff
@@ -11,6 +11,7 @@
     "@testing-library/jest-dom": "^5.15.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "aos": "^3.0.0-beta.6",
```

Comment:
It looks like you're now importing AOS this way and you are also importing it as a script tag in index.html. It should only be one or the other I think.